src/components/Notes/index.tsx

```diff
@@ -27,13 +29,20 @@ const Notes = () => {
   const [created, setCreated] = React.useState<boolean>(false);
   const [mounted, setMounted] = React.useState<boolean>(false);
 
+  const { id: patientId } = useParams();
+
+  const notDuplicatedArray = notesLocal.filter(
+    (note, index, self) => self.findIndex((n) => n.id === note.id) === index
+  );
+
+  const emptyArray = Array(4).fill(null);
```

Comment:
make it outside the component, as now you make initialisation with each rerender